### ISSUE TITLE
v6.0.5 <ArgsTable /> UI broken

### ISSUE THREAD

**Issue by shincurry** [storybookjs/storybook#11970]
**Describe the bug**

`<ArgsTable />` UI broken if description is too long.

**Screenshots**
![截屏2020-08-13 下午7 19 44](https://user-images.githubusercontent.com/4946624/90128748-f77b2900-dd99-11ea-9f30-90a5ffeaa6cb.png)

**System:**
```
Environment Info:

  System:
    OS: macOS 10.15.4
    CPU: (4) x64 Intel(R) Core(TM) i3-8100B CPU @ 3.60GHz
  Binaries:
    Node: 12.13.0 - ~/.nvm/versions/node/v12.13.0/bin/node
    Yarn: 1.22.4 - ~/dev/uui/node_modules/.bin/yarn
    npm: 6.12.0 - ~/.nvm/versions/node/v12.13.0/bin/npm
  Browsers:
    Chrome: 84.0.4147.125
    Firefox: 78.0.2
    Safari: 13.1
  npmPackages:
    @storybook/addon-a11y: 6.0.5 => 6.0.5
    @storybook/addon-controls: 6.0.5 => 6.0.5
    @storybook/addon-docs: 6.0.5 => 6.0.5
    @storybook/addon-storysource: 6.0.5 => 6.0.5
    @storybook/addons: 6.0.5 => 6.0.5
    @storybook/react: 6.0.5 => 6.0.5

```

**Comment by ajkl2533** [comment]
This should be fixed with #11868

**Comment by shincurry** [comment]
v6.0.13 fixed.
